recipes/aqueduct3/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "aqueduct3" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "1ebeba7f16cd4f14f1de833224f55b0cac7a8ef5f49344fa64d96b96948b6ffb" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package has a python entry point. Please add it here:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and test it in the `commands` section under `tests`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands